core/domain/collection_jobs_one_off_test.py

```diff
@@ -131,7 +131,11 @@ def test_migrate_colections_failing_strict_validation(self):
             'category': collection_category,
         }])
 
-        # Start migration job on sample collection.
+        collection_summary = collection_services.compute_summary_of_collection(
```

Comment:
Wait ... I think I'm missing something. Do we actually need to add this here, since the test is all about the collection object itself? Ditto below.